httputil/client.go

```diff
@@ -41,11 +155,20 @@ func NewHTTPClient(opts *ClientOptions) *http.Client {
 	}
 
 	transport := newDefaultTransport()
-	transport.TLSClientConfig = opts.TLSConfig
 	if opts.Proxy != nil {
 		transport.Proxy = opts.Proxy
 	}
 	transport.ProxyConnectHeader = http.Header{"User-Agent": []string{UserAgent()}}
+	// Remember the original ClientOptions.TLSConfig when making
+	// tls connection.
+	// Note that we only set TLSClientConfig here because it's extracted
+	// by the cmd/snap-repair/runner_test.go
```

Comment:
Does this comment mean that the assignment is otherwise meaningless?

Reply:
That is correct.